guess.py

```diff
@@ -0,0 +1,7 @@
+from random import randint
+
+nombre = input("Entrez un nombre…")
```

Comment:
```suggestion
nombre = input("Entrez un nombre: ")
```
I think tis is better with ":"